Lib/_collections_abc.py

```diff
@@ -409,6 +415,81 @@ def __subclasshook__(cls, C):
         return NotImplemented
 
 
+class _CallableGenericAlias(GenericAlias):
+    """ Internal class specifically for consistency between the ``__args__`` of
+    ``collections.abc.Callable``'s and ``typing.Callable``'s ``GenericAlias``.
+
+    See :issue:`42195`.
```

Comment:
Let's not make a habit of referencing issues from comments/docstrings. It is almost always better to just explain the concern, if it isn't clear from reading the source.